Dockerfile

```
@@ -0,0 +1,39 @@
+# Copy package files and install dependencies
+COPY package*.json ./
+RUN npm ci
+
+# Copy application source code
+COPY . .
+
+# Generate Prisma client
+RUN npm run prisma:generate
+
+# Build the application
+RUN npm run build
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)